packages/core/src/RenderPipeline/DepthOnlyPass.ts

```diff
@@ -56,6 +57,8 @@ export class DepthOnlyPass extends PipelinePass {
     engine._renderCount++;
     cullingResults.opaqueQueue.render(context, PipelineStage.DepthOnly);
     cullingResults.alphaTestQueue.render(context, PipelineStage.DepthOnly);
+    // Reset stencil
+    camera.scene._maskManager.drawMask(context, PipelineStage.Forward, SpriteMaskLayer.Nothing);
```

Comment:
Currently there is no need to do this part because `SpriteMaskShader` does not contain `depthOnlyPass`

Reply:
done